.happy/terraform/modules/batch/variables.tf

```diff
@@ -0,0 +1,46 @@
+variable stack_resource_prefix {
+  type        = string
+  description = "Prefix for account-level resources"
+}
+
+variable app_name {
+  type        = string
+  description = "Batch app name"
+}
```

Comment:
what do you mean by batch app name? Is this like the name of the compute env?

Reply:
It's the name of the batch workflow (nextstrain) and gets appended to the end of the batch job definition name.